src/Contrib/Otlp/MetricExporterFactory.php

```diff
@@ -67,6 +69,19 @@ private function buildTransport(string $protocol): TransportInterface
         );
     }
 
+    private function getTemporality(): string
+    {
+        $value = Configuration::getEnum(Variables::OTEL_EXPORTER_OTLP_METRICS_TEMPORALITY_PREFERENCE);
+        switch (strtolower($value)) {
+            case strtolower(Temporality::CUMULATIVE):
```

Comment:
Should use literals, all usages of `Temporality` should treat these constants as [`string|Temporality`](https://github.com/open-telemetry/opentelemetry-php/blob/1976f53f92fabdcd7f53f83fccacfb95fde007de/src/SDK/Metrics/Data/Temporality.php#L10) to allow converting the class to an enum once support for 8.0 is dropped.